internals/cli/cmd_changes.go

```diff
@@ -24,15 +24,8 @@ import (
 	"github.com/canonical/pebble/client"
 )
 
-var shortChangesHelp = "List system changes"
-var shortTasksHelp = "List a change's tasks"
-var longChangesHelp = `
-The changes command displays a summary of system changes performed recently.
-`
-var longTasksHelp = `
-The tasks command displays a summary of tasks associated with an individual
-change that happened recently.
-`
+const cmdChangesSummary = "List system changes"
+const cmdChangesDescription = "The changes command displays a summary of system changes performed recently."
```

Comment:
Can we please consistently break the description into the next line with backticks as we (mostly?) had before.